Android build broken

After merging #17 , CI stopped working for Android builds. See https://github.com/2060-io/hologram-app/actions/runs/11282960830/job/31382655878.

```
> Task :react-native-worklets-core:buildCMakeRelWithDebInfo[x86_64]
> Task :react-native-worklets-core:externalNativeBuildRelease
> Task :react-native-worklets-core:prefabReleasePackage

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-vision-camera:buildCMakeRelWithDebInfo[arm64-v8a]'.
> com.android.ide.common.process.ProcessException: ninja: Entering directory `/Users/admin/actions-runner/_work/hologram-app/hologram-app/node_modules/react-native-vision-camera/android/.cxx/RelWithDebInfo/g5x1g1e3/arm64-v8a'
  
  C++ build system [build] failed while executing:
      /Users/admin/Library/Android/sdk/cmake/3.22.1/bin/ninja \
        -C \
        /Users/admin/actions-runner/_work/hologram-app/hologram-app/node_modules/react-native-vision-camera/android/.cxx/RelWithDebInfo/g5x1g1e3/arm64-v8a \
        VisionCamera
    from /Users/admin/actions-runner/_work/hologram-app/hologram-app/node_modules/react-native-vision-camera/android
  ninja: error: '/Users/admin/actions-runner/_work/hologram-app/hologram-app/node_modules/react-native-worklets-core/android/build/intermediates/cmake/debug/obj/arm64-v8a/librnworklets.so', needed by '../../../../build/intermediates/cxx/RelWithDebInfo/g5x1g1e3/obj/arm64-v8a/libVisionCamera.so', missing and no known rule to make it
```


Locally, it does not work for me neither when doing the same steps of CI:
```
- yarn install
- cd android && ./gradlew clean assembleDevDebug
```

Note: in CI we use assembleDevRelease, but upload keys are needed, so this is a way to reproduce the issue in any computer.

--- a/patches/react-native-vision-camera+4.5.2.patch
+++ b/patches/react-native-vision-camera+4.5.2.patch
@@ -1,12 +1,24 @@
 diff --git a/node_modules/react-native-vision-camera/android/CMakeLists.txt b/node_modules/react-native-vision-camera/android/CMakeLists.txt
-index 1460dbd..c100951 100644
+index 1460dbd..49ab4b6 100644
 --- a/node_modules/react-native-vision-camera/android/CMakeLists.txt
 +++ b/node_modules/react-native-vision-camera/android/CMakeLists.txt
-@@ -62,6 +62,7 @@ target_link_libraries(
+@@ -15,6 +15,11 @@ find_library(LOG_LIB log)
+ add_definitions(-DEGL_EGLEXT_PROTOTYPES)
+ add_definitions(-DGL_GLEXT_PROTOTYPES)
+ 
++if (${CMAKE_BUILD_TYPE} MATCHES "Debug")
++        set(BUILD_TYPE "debug")
++else()
++        set(BUILD_TYPE "release")
++endif()
+ if (ENABLE_FRAME_PROCESSORS)
+         add_definitions(-DVISION_CAMERA_ENABLE_FRAME_PROCESSORS=true)
+ else()
+@@ -62,6 +67,7 @@ target_link_libraries(
          ReactAndroid::jsi                   # <-- RN: JSI
          ReactAndroid::reactnativejni        # <-- RN: React Native JNI bindings
          fbjni::fbjni                        # <-- fbjni
-+        "${NODE_MODULES_DIR}/react-native-worklets-core/android/build/intermediates/cmake/debug/obj/${ANDROID_ABI}/librnworklets.so"
++        "${NODE_MODULES_DIR}/react-native-worklets-core/android/build/intermediates/cmake/${BUILD_TYPE}/obj/${ANDROID_ABI}/librnworklets.so"
  )
  
  # Optionally also add Frame Processors here

